presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergSplitManager.java

```diff
@@ -180,7 +180,7 @@ private void validateSplitsPlannedForSql(SplitManager splitManager,
     {
         Session session = sessionWithFilterPushdown(filterPushdown);
         List<TableScanNode> tableScanNodes = getTableScanFromOptimizedPlanOfSql(sql, session);
-        assertTrue(tableScanNodes != null && tableScanNodes.size() == 1);
+        assertEquals(tableScanNodes.size(), 1);
```

Comment:
This change will cause the original `AssertionError` changing to a `NPE` when `tableScanNodes` happens to be null. Feels like it went from an expected error to an unexpected one. Is that okay?

Reply:
That's fine. In both cases the test fails, which is what we want. Now it's more obvious why the test failed: wrong number of nodes or null nodes, so there's more information.

Reply:
Rather than throwing an NPE due to `tableScanNodes` being null I think it would be better to add an `assertNotNull` beforehand as it will be clearer to future authors that this value must be non-null

Reply:
done